server/src/routes/Authorize.ts

```diff
@@ -1,99 +1,103 @@
+         */
 
-        /* DEVELOPER NOTES:
-        * Here we will use the token to get the EoB data for the mocked 'user' of the sample application
-        * then to save trips to the BB2 API we will store it in the mocked db with the mocked 'user'
-        *
+    // Here we are grabbing the mocked 'user' for our application
+    // to be able to store the access token for that user
+    // thereby linking the 'user' of our sample applicaiton with their Medicare.gov account
+    // providing access to their Medicare data to our sample application
+    const loggedInUser = getLoggedInUser(db);
+    loggedInUser.authToken = authToken;
+
+    /* DEVELOPER NOTES:
+        * Here we will use the token to get the EoB data for the mocked 'user' of the sample
+        * application, then to save trips to the BB2 API we will store it in the mocked db
+        * with the mocked 'user'
         * You could also request data for the Patient endpoint and/or the Coverage endpoint here
         * using similar functionality
         */
-        const eobData = await getBenefitData( req, res);
-        loggedInUser.eobData = eobData;
-
-    } catch (e) {
-        /* DEVELOPER NOTES:
+    const eobResp = await getBenefitData(req);
+    loggedInUser.eobData = eobResp.data;
+  } catch (e) {
+    /* DEVELOPER NOTES:
         * This is where you could also use a data service or other exception handling
         * to display or store the error
-         */
-        console.log(e);
-    }
-        /* DEVELOPER NOTE:
+        */
+    logger.err(e);
+  }
+  /* DEVELOPER NOTE:
          * This is a hardcoded redirect, but this should be used from settings stored in a conf file
          * or other mechanism
          */
-    res.redirect(`http://localhost:3000`);
+  res.redirect('http://localhost:3000');
 }
 
 export async function getAuthUrl(req: Request, res: Response) {
-
-    /* DEVELOPER NOTE:
+  /* DEVELOPER NOTE:
     * to utilize the latest security features/best practices
     * it is recommended to utilize pkce
     */
-    const pkce = req.params.pkce === 'true';
-    db.settings = new Settings({
-        version: req.query?.version?.toString() || db.settings.version,
-        env: req.query?.env?.toString() || db.settings.env,
-        pkce: req.query?.pkce?.toString() ? pkce : db.settings.pkce
-    });
-    res.send(generateAuthorizeUrl());
+  const pkce = req.params.pkce === 'true';
+  db.settings = new Settings({
+    version: req.query?.version?.toString() || db.settings.version,
+    env: req.query?.env?.toString() || db.settings.env,
+    pkce: req.query?.pkce?.toString() ? pkce : db.settings.pkce,
+  });
+  res.send(generateAuthorizeUrl());
 }
 
 export async function getCurrentAuthToken(req: Request, res: Response) {
-    const loggedInUser = getLoggedInUser(db);
-    res.send(loggedInUser.authToken);
+  const loggedInUser = getLoggedInUser(db);
+  res.send(loggedInUser.authToken);
 }
 
 const router = Router();
 
+// eslint-disable-next-line
```

Comment:
Why disabling eslint here?

Reply:
changed to more specific disable
// eslint-disable-next-line @typescript-eslint/no-misused-promises

the reason is the router get takes an async func as 2nd parameter - that return a promise where a void is expected.